pkg/benchmarks/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+module.exports = {
+  env: {
+    commonjs: true
+  },
+  parserOptions: {
+    ecmaVersion: 2019
```

Comment:
this is a literal explicit breaking of backwards compatability.

this is what you said you weren't going to do.

Reply:
All of the code that will be run on non-node12 will be run through babel first, so this will have no effect on backwards-compat.

Reply:
Oh god babel?

1. A lot of babel's shims are just incorrect
2. You can expect the library to slow down ~300%
3. Babel introduces a lot of undefined behavior instead of throwing when it hits undefined behavior.  These kinds of bugs are essentially impossible to locate
4. If it's being transcompiled, can we use `typescript` instead?  
    1. We can start with all checks off, so it won't slow you down
    2. It will immediately add significant value in noticing things like missing arguments
    3. Its shims are far faster and much more commonly correct
    4. You get much better language support
    5. We have the option of stepping into a typesystem progressively with time
    6. We need it anyway for dropping .d.ts
    7. Everybody can help (babel is an increasingly rare skill outside the react ecosystem, and the react ecosystem is mostly moved to typescript at this point)

&nbsp;

> this will have no effect on backwards-compat.

Babel's shims clash violently with modernizr and a lot of peg people use modernizr